### ISSUE TITLE
[Feature request] Move to MIT License

### ISSUE THREAD

**Issue by geofurb** [coqui-ai/TTS#3587]
The company is shutting down and can no longer license this project for commercial purposes or benefit from such licensing. Suggest moving to MIT license for more permissive modification and redistribution by the community.

**Comment by mahimairaja** [comment]
++

**Comment by arnaublanco** [comment]
I agree.

**Comment by daemon** [comment]
+1

I don't mean to jump to conclusions, but I hope the founders don't develop a "ship dies with the captain" mentality that happens so frequently.

**Comment by rose07** [comment]
This website is also owned by Microsoft. You can give it a try

https://tts.byylook.com/ai/text-to-speech

**Comment by metal3d** [comment]
I completely agree. The model and the use of the model is a bit too restrictive now the enterprise is shutting down.

BTW: Is there any page explaining why they stopped? And if the project may live with the community?

**Comment by geofurb** [comment]
Wanted to re-up this, @erogol @JRMeyer @kdavis-coqui

**Comment by eginhard** [comment]
The codebase is licensed under [MPL 2.0](https://github.com/coqui-ai/TTS/blob/dev/LICENSE.txt) which is fairly permissive and allows commercial use already. As for the XTTS model license, [previous](https://github.com/coqui-ai/TTS/issues/3490#issuecomment-1878315151) [comments](https://github.com/coqui-ai/TTS/issues/3490#issuecomment-1878982722) indicate that it's unlikely to change.

**Comment by geofurb** [comment]
Bump

**Comment by mantrakp04** [comment]
Any updates, even the link to purchase one is not working

**Comment by daemon** [comment]
The (unofficial) legal opinion that I'm aware of is that these models can be treated as public domain, based on [these answers](https://github.com/coqui-ai/TTS/issues/3490#issuecomment-1878982722):

> Who will own the IP of Coqui (& licensing rights) after the shutdown?
> Answer: noone

**Comment by mantrakp04** [comment]
> The (unofficial) legal opinion that I'm aware of is that these models can be treated as public domain, based on [these answers](https://github.com/coqui-ai/TTS/issues/3490#issuecomment-1878982722):
> 
> > Who will own the IP of Coqui (& licensing rights) after the shutdown?
> > Answer: noone

So as of now can we consider xtts to be under public domain? I mean its still very unclear tho, i hope smone from the coqui team clarify this

**Comment by dhoesele** [comment]
Bump

**Comment by arnaublanco** [comment]
There's nothing we can do :(

**Comment by metal3d** [comment]
So here is an activity, damned bot

**Comment by medemi68** [comment]
Yeah this is getting ridiculous. If they’re abandoning it, they need to make it clear. It’s pretty cowardly to do what they’re doing right now.

**Comment by eginhard** [comment]
The licenses are already clearly stated (MPL 2.0 for the code, CPML for the XTTS model). I wouldn't expect these to change since the company has shut down.

**Comment by medemi68** [comment]
> The licenses are already clearly stated (MPL 2.0 for the code, CPML for the XTTS model). I wouldn't expect these to change since the company has shut down.

I get that, but they really should open source the model or make it more permissive. I'm actually kind of surprised that they were able to even sell commercial licenses since I think some of the training data is CC-BY-NC 4.0. I can't remember which one I saw that mentioned that.

**Comment by MikeRich88** [comment]
Pre-emptive 🖕🖕🖕 for stale bot.
